src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php

```diff
@@ -50,18 +50,19 @@ class SnippetControllerTest extends SuluTestCase
     public function setUp()
     {
         parent::setUp();
-        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
-        $this->phpcrSession = $this->getContainer()->get('sulu_test.doctrine_phpcr')->getConnection();
         $this->initPhpcr();
-        $this->loadFixtures();
         $this->client = $this->createAuthenticatedClient();
+        $this->phpcrSession = $this->getContainer()->get('sulu_test.doctrine_phpcr')->getConnection();
+        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
+        $this->loadFixtures();
     }
 
     /**
      * @dataProvider provideGet
      */
     public function testGet($locale, $expected)
     {
+        $this->documentManager->clear();
```

Comment:
Why do we need this line only for this test?

Reply:
It seems the only test which access the same document in different languages but will move it to the loadFixtures method so if new test are added not another side effects happens.